lib/puppet/provider.rb

```diff
@@ -239,6 +239,8 @@ def initialize(name, path, confiner)
       @optional = false
       @confiner = confiner
       @custom_environment = {}
+      @command_resolver = Puppet::Util
```

Comment:
The `Puppet::Util` module is [mixed into the provider](https://github.com/puppetlabs/puppet/blob/master/lib/puppet/provider.rb#L38), so I don't think this is necessary.